### PR TITLE
Fixes #15205 - Useless backquotes, sql preview, delete index

### LINKED ISSUE #15205: [UI] Useless backquotes, sql preview, delete index
### Describe the bug

Useless backquotes in the sql query: around the query and the table name (2 instead of one) 
### To Reproduce

1. Try to delete an index
2. See backquotes 

### Expected behavior

No quotes and a beautiful sql preview box

### Screenshots

Screenshot_20190415-221441

### Server configuration

- phpMyAdmin version: 4.8.6

### PR DESCRIPTION
**Pull request by saurass**
Signed-off-by: Saurabh Srivastava <saurabhsrivastava312@gmail.com>

### Description
Fixes #15205 

## Problem
* Pretty SQL not showing because we were using simple `PMA_confirm`
* The SQL query had backquotes because when sanitizing the query (line 754 `libraries/classes/Index.php`) we weren't passing `add_backquotes` to `false`.

## Solution
* Introduced a new function to `functions.js` --> `PMA_confirmPreviewSQL(....)`.
* Calling this function to show pretty SQL query in `indexes.js` line 602.
* Passing `add_backquotes` parameter as `false` (line 754 `libraries/classes/Index.php`) in function call - `Sanitize::jsFormat(....)`

This seems to fix our issue.

## Contribution Checklist
- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by saurass** [comment]
@sudo-bot Deploy pr

File: js/functions.js [hunk 1]
```diff
@@ -2407,6 +2407,52 @@ function PMA_previewSQL ($form) {
...
+ * @param callback  callback function
+ *
+ * @return void
+ */
+function PMA_confirmPreviewSQL (sql_data, url, callback) {
+    var $dialog_content = $('<div class="preview_sql"><code class="sql"><pre>'
+        + sql_data
+        + '</pre></code></div>'
+    );
+    var button_options = {};
```
Comment (williamdes) [inline js/functions.js]: can you remove this line

File: js/functions.js [hunk 2]
```diff
@@ -2407,6 +2407,52 @@ function PMA_previewSQL ($form) {
...
+        {
+            text: PMA_messages.strOK,
+            'class': 'submitOK',
+            click: function () {
+                callback(url);
+            }
+        },
+        {
+            text: PMA_messages.strCancel,
+            'class': 'submitCancel',
```
Comment (williamdes) [inline js/functions.js]: why did you use quotes for this property ?

**Review by williamdes — APPROVED** [review]
LGTM

File: js/functions.js (hunk 2 above)
Comment (saurass) [reply js/functions.js]: Not exactly sure why I wrote it. May be just in a flow :sweat_smile:

I will remove this

File: js/functions.js (hunk 1 above)
Comment (saurass) [reply js/functions.js]: Remove all these 4 lines ?

Comment (saurass) [reply js/functions.js]: I don't think line 2423 should be removed. It is important to make sure that `button_options` variable is empty before use.

About the line above that (2420-2423), these are to generate the SQL preview div.

Comment (williamdes) [reply js/functions.js]: @saurass only `2423`  `var button_options = {};` is useless since `var button_options = [` declares another time the variable

Comment (saurass) [reply js/functions.js]: Yeah okay, sure it can be removed .

**Review by williamdes — CHANGES_REQUESTED** [review]
Works fine for delete primary key but not for and index
Can you please have a look @saurass ?
Delete index screenshot
Delete PK screenshot

**Comment by saurass** [comment]
> Works fine for delete primary key but not for and index
> Can you please have a look @saurass ?
 Yeah sure, I will try to fix this.

**Comment by saurass** [comment]
@sudo-bot Deploy pr

**Comment by saurass** [comment]
I missed removing backquotes in `libraries/classes/Index.php` for `INDEX` keys. 

Should work well now @williamdes

**Review by williamdes — APPROVED** [review]
Merged, great work on this issue @saurass !

**Comment by saurass** [comment]
Thank you @williamdes

**Pull request merged**